lib/charms/opensearch/v0/constants_charm.py

```diff
@@ -112,3 +112,14 @@
 
 # User-face Backup ID format
 OPENSEARCH_BACKUP_ID_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+# Roles that we can add as tags to prometheus data
+COS_TAGGABLE_ROLES = [
+    "data",
+    "cluster_manager",
+    "voting",
+    "coordinating_only",
+    "ingest",
+    "cluster_manager_elegible",
```

Comment:
Ouch! I have an incoming next PR to reshuffle grafana dashboards order. I will also remove this unneeded entry there.

Reply:
I think you can directly use `ClusterTopology.generated_roles()`

Reply:
@Mehdi-Bendriss added via this PR: https://github.com/canonical/opensearch-operator/pull/332